frontend/packages/knative-plugin/src/components/overview/__tests__/overview-knative-test-data.ts

```diff
@@ -27,3 +27,23 @@ export const mockRoutesData = [
     },
   },
 ];
+
+export const mockEventSourceData = {
```

Comment:
can we have `type` of mockEventSourceData?

Reply:
done